Makefile

```diff
@@ -132,6 +132,8 @@ manifests: controller-gen yq
 	cp config/crd/minimal/* charts/kserve-crd-minimal/templates/
 	rm charts/kserve-crd-minimal/templates/kustomization.yaml
 
+	kubectl kustomize config/crd/external/gie > config/llmisvc/gie.yaml
```

Comment:
do we need InferencePool as well? or only InferenceModel enough

Reply:
We need both